go/systests/profile_test.go

```diff
@@ -0,0 +1,160 @@
+package systests
+
+import (
+	"context"
+	"testing"
+
+	"github.com/davecgh/go-spew/spew"
+	"github.com/keybase/client/go/engine"
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+	"github.com/stretchr/testify/require"
+)
+
+func TestProofSuggestions(t *testing.T) {
```

Comment:
This test is picky and will make reconfiguring the dev server more annoying. Curious if you think there's another way.

Reply:
the externals test is kinda in the same state which was annoying.. it seemed to stabilize though i like the explicitness of these checks